internal/specs/compliance/nsa-1.0.yaml

```diff
@@ -0,0 +1,175 @@
+---
```

Comment:
For my understanding, how was this spec created? Is it available somewhere as a yaml file or you just translated their PDF into this yaml?

Would you know if they have similar spec for AWS?

Reply:
I have created it my self bases on investigation of the NSA spec and made the relevant mapping to rego